.github/workflows/utilities-pulumi_state_splitter.yml

```diff
@@ -0,0 +1,24 @@
+name: pulumi_state_splitter
+
+on:
+  push:
+    paths:
+      - .github/workflows/utilities-pulumi_state_splitter.yml
+      - utilities/pulumi_state_splitter/**
+
+jobs:
+  tests:
+    runs-on: ubuntu-latest
+    name: tests
+    steps:
+      - uses: actions/checkout@master
+      - uses: actions/setup-python@v5
+        with:
+          python-version: '3.11'
+      - uses: actions/cache@v3
+        with:
+            path: ~/.cache/pypoetry
+            key: ${{ hashFiles('utilities/pulumi_state_splitter/poetry.lock') }}
```

Comment:
Indentation is off:
```suggestion
        with:
          path: ~/.cache/pypoetry
          key: ${{ hashFiles('utilities/pulumi_state_splitter/poetry.lock') }}
```

Reply:
Done in cb4d7925f71be2487ea8ddb93d28130d2c1064e2 .